app/middleware/stripe-setup/restrict-to-live-stripe-account.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = (req, res, next) => {
```

Comment:
This middleware could do with potentially a few names/ comments to make this 
* easier to read 
* provide more complete information during a stack trace

Suggestions: 
```js
const restrictRequestsToLiveAccounts = function restrictRequestsToLiveAccounts (req, res, next) { 
  const requestHasValidLiveAccount = req.account && req.account.type // ... 

  if (requestHasValidLiveAccount) { 
    next()
  } else { 
    // request is missing information - a 404 Not Found should be displayed? 
  }
}
module.exports = restrictRequestsToLiveAccounts

Reply:
Thanks, I have added `function` to both of the middlewares
and comment explaining why the error should be `404`